config/settings.yml

```diff
@@ -12,6 +12,7 @@ saml:
   issuer: saml-rp.vetsgov.localhost
   callback_url: http://localhost:3000/auth/saml/callback
   metadata_url: https://api.idmelabs.com/saml/metadata/provider
+  authn_requests_signed: false
```

Comment:
Even though I've created the devops PR to override this to `true`, ( https://github.com/department-of-veterans-affairs/devops/pull/6488 ) I'm tempted to have it `true` here as well and force all local devs to override it to `false` in their respective `settings.local.yml` files, thoughts?

Reply:
It'd be more annoying for local devs, but also seems more appropriate since this setting would only ever be disabled for localhost authentication.

Reply:
Let's err on best for developers!